### PR TITLE
Upgrade Decidim to 0.26

### PR DESCRIPTION
**Pull request by NicoleSo13**
This PR upgrades the application to Decidim version 0.26.
Release notes: https://github.com/decidim/decidim/releases/tag/v0.26.0

On deploy it will require to execute the following commands:

- Upgrade server's Ruby version to 2.7.5
- For locked users are present in global search, to update the search and make them disappear, Run in a bin/rails console or create a migration with:
  `Decidim::User.find_each(&:try_update_index_for_search_resource)`
- For fixing statistic in comments. There's a rake task that you need to run:
  `bin/rails decidim_comments:update_participatory_process_in_comments`
- For Base64 images migration run the following task:
  `bin/rails decidim:active_storage_migrations:migrate_inline_images_to_active_storage[admin_email]`

### CODE REVIEW

**Review by tramuntanal — APPROVED** [review]
LGTM @NicoleSo13 

But you comment in the description that whe must Upgrade server's Ruby version to 2.7.5, but in master we already have Ruby 2.7.5. So this PR is not changing Ruby version. Also I've checked and there is Ruby 2.7.5 in the server.

File: Gemfile [hunk 1]
```diff
@@ -4,17 +4,17 @@ source "https://rubygems.org"
...
-gem "decidim-file_authorization_handler", git: "https://github.com/CodiTramuntana/decidim-file_authorization_handler.git", branch: "release/0.25-stable"
+gem "decidim-file_authorization_handler", git: "https://github.com/CodiTramuntana/decidim-file_authorization_handler.git", branch: "release/0.26-stable"
 gem "decidim-initiatives", DECIDIM_VERSION
 gem "decidim-members", git: "https://github.com/CodiTramuntana/decidim-members.git", tag: "v0.1.19"
 gem "decidim-sortitions", DECIDIM_VERSION
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer"
+gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer", branch: "release/0.26-stable"
 gem "decidim-verifications-csv_email", git: "https://github.com/CodiTramuntana/decidim-verifications-csv_emails.git", tag: "v0.0.11"
-gem "decidim-verifications-sant_boi_census", git: "https://github.com/CodiTramuntana/decidim-verifications-sant_boi_census.git", tag: "v0.1.0"
+gem "decidim-verifications-sant_boi_census", git: "https://github.com/CodiTramuntana/decidim-verifications-sant_boi_census.git", branch: "upgrade-decidim-to-0-26"
```
Comment (tramuntanal) [inline Gemfile:17]: We must wait for this gem to have the final version before merging: tag 0.1.3

**Review by tramuntanal — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (NicoleSo13) [reply Gemfile:17]: Es que no hay tag 0.1.3 de esta gema, el último es 0.1.0

Comment (laurajaime) [reply Gemfile]: @tramuntanal there is no 1.3.0 tag, I think the branch is being used in other places too.

Comment (NicoleSo13) [reply Gemfile]: I have created 0.1.3 tag recently on decidim-verifications-sant_boi_census

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**